engine_sparse_data.ipynb

```diff
@@ -1038,16 +992,25 @@
     }
    ],
    "source": [
-    "random_value = df.target.value_counts(normalize=True).max()\n",
+    "majority_class_accuracy = df.target.value_counts(normalize=True).max()\n",
     "\n",
     "ax = prediction_stats_df.plot()\n",
     "ax.set_ylim(-0.05, 1.05)\n",
-    "ax.hlines(y=random_value, xmin=0, xmax=0.9, color=\"black\", label=\"Random Prediction\")\n",
+    "ax.hlines(y=majority_class_accuracy, xmin=0, xmax=0.9, color=\"black\", label=\"Majority Class Prediction\")\n",
     "ax.legend(loc=\"lower right\")\n",
     "ax.set_title(\"Prediction Stats for Different Levels of Sparsity\")\n",
     "\n",
     "plt.savefig(\"data/sparse_data_plot.png\")"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Conclusion\n",
+    "\n",
+    "In this notebook we demonstrate the ability of the Howso Engine to natively handle data with missing values. Furthermore we demonstrate that the Howso Engine can maintain accuracy even when the training data has increasing proportions of missing values. This makes the Howso Engine a great tool to use in use-cases where datasets have many missing values (even when the missing values themselves are informative!)."
```

Comment:
Could we move this blurb to the top, it seems like it will easier for users to determine if this recipe is useful for them, if we tell them at the beginning what it does.